pymc_marketing/mmm/tvp.py

```diff
@@ -185,7 +178,7 @@ def create_time_varying_gp_multiplier(
     time_index: pt.sharedvar.TensorSharedVariable,
     time_index_mid: int,
     time_resolution: int,
-    model_config: dict,
+    model_config: dict | HSGPKwargs,
```

Comment:
Lets just use `hsgp_kwargs` instead here and pass it in from the delayed_saturated_mmm use. Then this function is unaware of our convention of the key names in the model config

```python
# In delayed saturated use
media_latent_process = create_time_varying_gp_multiplier(
    name="media",
    dims="date",
    time_index=time_index,
    time_index_mid=self._time_index_mid,
    time_resolution=self._time_resolution,
    hsgp_kwargs=self.model_config["media_tvp_config"],
)
```

Reply:
Good point! added in https://github.com/pymc-labs/pymc-marketing/pull/809/commits/86e4d7a17b036054542350a2d68f1f9e7c64be60